### PR TITLE
The beginnings of a template FAQ

### PR DESCRIPTION
**Pull request by devbytyler**
Adds answers to these questions:
- How do I populate "Nature of Agreement" found in Leveraged Authorizations and Interconnections tables?
- How do I model a "Hybrid" or "Shared" control origination?
- How can I get the table of contents to print?

### CODE REVIEW

File: docs/documents/faq.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+In a perfect world, everybody might be adept at producing and consuming OSCAL. In reality, human-readable documentation is still a necessity and each vendor represents their content differently. Consequently, it can be challenging to fit existing documentation into the OSCAL model while still maintaining consistent outputs.
+
+This FAQ will help you navigate common questions and issues related to specific documents.
+
+## FedRAMP
+
+### SSP
+
+::: details How do I populate "Nature of Agreement" found in Leveraged Authorizations and Interconnections tables?
```
Comment (MikeWillCook) [inline docs/documents/faq.md:27]: Worth adding "... found in Section #"?

Comment (devbytyler) [reply docs/documents/faq.md:27]: great idea

**Comment by joshdalton** [comment]
@devbytyler on the shared and hybrid concept, is it accurate to say that for each responsible role there would need to be separate Risk Solution? Do you want to touch on that concept?

File: docs/documents/faq.md (hunk 1 above)
Comment (devbytyler) [reply docs/documents/faq.md:27]: Gets a bit tricky with Rev4 and Rev5. And the question is for two different sections.

**Review by MikeWillCook — APPROVED** [review]

File: docs/documents/faq.md (hunk 1 above)
Comment (devbytyler) [reply docs/documents/faq.md:27]: How's that for now?

image

**Comment by devbytyler** [comment]
@joshdalton Added a sentence 
"Risk Solutions are **single origination** by design. Multi-responsibility Risk Solutions should be broken up for best results.
If you are converting legacy documentation into custom control responses, you can use a combination of originations to model a hybrid or shared control."

**Review by joshdalton — APPROVED** [review]

**Pull request merged**